### ISSUE TITLE
show console output in console ui

### ISSUE THREAD

**Issue by HeladoDeBrownie** [HeladoDeBrownie/Nexus#4]
currently output from running lua code in the console is sent only to standard output. it must be shown in the console instead or in addition

**Comment by HeladoDeBrownie** [comment]
partly addressed by cc7c214a0c450e1c6b83c0af524bc9a5fabe94c2 and by 3f83251c3935db105388e351612983d9acaef0ff

**Comment by HeladoDeBrownie** [comment]
almost completely addressed by 9901fb413ea183b6b847cc5d687395be1804ac34. now all that's missing is printing the results of calls, and possibly bare expression evaluation

**Comment by HeladoDeBrownie** [comment]
both of these final items are now completely resolved by de3e9fec0194cd2a404af650a53924164928b9dd!
